goose3/article.py

```diff
@@ -49,6 +49,8 @@ def __init__(self):
         self._doc = None
         self._raw_doc = None
         self._publish_date = None
+        self._publish_datetime = None
+        self._publish_utc = None
```

Comment:
I think `publish_datetime_utc` is more descriptive.

Reply:
I agree, I will make those changes tonight after work!